### ISSUE TITLE
Windows MSR driver compilation error

### ISSUE THREAD

**Issue by KylePan-01** [intel/pcm#665]
I use Visual Studio Community 2022,  Windows SDK 10.0.22621.2428 and WDK 10.0.22621.2428 to compile MSR in the pcm/src/WinMSRDriver, but I got the following error:
```
“C:\Users\34957\Documents\GitHub\pcm\src\WinMSRDriver\MSR.vcxproj”(默认目标) (1) ->
(ValidateNTTargetVersion 目标) ->
  C:\Program Files (x86)\Windows Kits\10\build\10.0.22621.0\WindowsDriver.common.targets(314,5): error MSB4044: 未给任务“ValidateNTTargetVersion”的必需参数“ValidNTTargetVersions”赋值。 [C:\Users\34957\Documents\GitHub\pcm\src\WinMSRDriver\MSR.vcxproj]
```
It means that "ValidNTTargetVersions" not given, but I can not find this parameter in MSR.vcxproj.
Does it caused by VS 2022? Because I read the compilation in the Windows Build Pass, which used VS 2019 rather than 2022.

**Comment by phtephanx** [comment]
Related: https://github.com/intel/pcm/issues/615

I get the same error also using VS Community 2022.

**Comment by rdementi** [comment]
> I made the changes suggested here : https://learn.microsoft.com/en-us/answers/questions/1526163/msb4044-issue-compiling-msr-driver-and-generating and my MSR compile succeeded

thanks a lot for sharing the solution. We will adopt it asap.

**Comment by rdementi** [comment]
this issue should now be resolved in the master branch : https://github.com/intel/pcm/commit/56901851deda8a6f78254fa9fbef3d4c3afdf7a4

**Comment by rdementi** [comment]
please reopen if the issue is still there

**Comment by MatteoBax** [comment]
> this issue should now be resolved in the master branch : [5690185](https://github.com/intel/pcm/commit/56901851deda8a6f78254fa9fbef3d4c3afdf7a4)

The problem has been solved, thanks.
